### PR TITLE
Add indirect specular section

### PR DESCRIPTION
**Pull request by aevyrie**

### CODE REVIEW

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 1]
```diff
@@ -810,9 +810,23 @@ TODO.
...
+effect tends to make surfaces behave like mirrors. This over-brightening happens because the
+surfaces must reflect *something*, but without path traced or screen-space reflections, the renderer
+has to guess *what* is being reflected. The best guess it can make is to sample the environment cube
+map, even if light would've hit something else before reaching the environment light. This artifact
+where light occlusion is ignored is called specular light leaking.
+
+Consider a car tire; though the rubber might be shiny, you wouldn't expect it to have bright
+specular highlights inside a wheel well, because the car itself is blocking (occluding) the light
+that would otherwise cause these reflections. Checking for occlusion can be computationally
+expensive. Instead, this change uses bevy's existing screen space ambient occlusion to approximate
```
Comment (TrialDragon) [inline content/news/2024-02-03-bevy-0.13/index.md:825]:
```suggestion
expensive. Instead, this change uses Bevy's existing screen space ambient occlusion to approximate
```

**Review by TrialDragon — CHANGES_REQUESTED** [review]
Minor typo fix, otherwise looks good to me grammar-wise.

**Comment by alice-i-cecile** [comment]
I'd really like to include your motorcycle image in here: it's a really clear demonstration of the problem and the fix. Plus it's pretty!

**Review by TrialDragon — APPROVED** [review]

**Comment by aevyrie** [comment]
Copying discord message here:

> after adding attribution for the motorcycle model for 0.13, I hit a strange snag
the model I downloaded was CC BY 4.0
But the link to the model, now lists it as a Standard (paid) license.
It looks like I stumbled across the model when it was uploaded, downloaded it, then they changed the license.
The license.txt included with the file explicitly says the model is CC BY 4.0, but I'm not sure if that matters in practice.

**Review by alice-i-cecile — APPROVED** [review]
I'm comfortable using this image: we're not redistributing the underlying model.

**Pull request merged**